api/src/repositories/system-constant.ts

```diff
@@ -0,0 +1,56 @@
+import { getKnex } from '../database/db';
+import { BaseRepository } from './base-repository';
+
+export interface ISystemConstant {
+  system_constant_id: number;
+  constant_name: string;
+  character_value: string;
+  numeric_value: number;
+  description: string;
+  create_date: string;
+  create_user: number;
+  update_date: string | null;
+  update_user: number | null;
+  revision_count: number;
+}
+
+/**
+ * A repository class for accessing system constant records.
+ *
+ * @export
+ * @class SystemConstantRepository
+ * @extends {BaseRepository}
+ */
+export class SystemConstantRepository extends BaseRepository {
+  /**
+   * Fetch one system constant by name.
+   *
+   * @param {string} constantName
+   * @return {*}  {Promise<ISystemConstant>}
+   * @memberof SystemConstantRepository
+   */
+  async getSystemConstant(constantName: string): Promise<ISystemConstant> {
+    const knex = getKnex();
+    const queryBuilder = knex.queryBuilder().select().from('system_constant').where('constant_name', constantName);
+
+    const response = await this.connection.knex<ISystemConstant>(queryBuilder);
+
+    return response.rows[0];
```

Comment:
Needs case when `rows === undefined`?

Reply:
Same for `getSystemConstants`

Reply:
`rows` will always be an array, so accessing the 0th index is safe, it will just return `undefined` if there are no elements.

That being said, this function is actually never used, so I am just going to remove it.